toolkit/tools/imagepkgfetcher/imagepkgfetcher.go

```diff
@@ -88,7 +88,7 @@ func main() {
 		enabledRepos = enabledRepos & ^rpmrepocloner.RepoFlagUpstream
 	}
 	if *disableDefaultRepos {
-		enabledRepos = enabledRepos & ^rpmrepocloner.RepoFlagMarinerDefaults
+		enabledRepos = enabledRepos & ^rpmrepocloner.RepoFlagAzureLinuxDefaults
```

Comment:
consider replacing AzureLinux with Distro here (if you do it in the other places)